src/components/Loading/index.js

```diff
@@ -1,9 +1,43 @@
 import React from 'react';
+import CircleProgressBar from 'components/CircleProgressBar';
+import styles from './loading.scss';
 
-const Loading = ({ isLoading, children, onServer }) => (
-  onServer || (__CLIENT__ && isLoading) || !children
-    ? <div>Loading screen</div>
-    : children
-);
+const Loading = React.createClass({
+
+  getInitialState () {
+    return { percentage: 75 };
+  },
+
+  tick () {
+    this.setState({ percentage: this.state.percentage + 125 });
```

Comment:
@mattberridge I'm all for a CSS-only solution instead. This was a band-aid since I saw using CircleProgressBar as a quick fix. 

Initial percentage of `75` then then adding `125` every second just staggers the animation based on what CircleProgressBar can already do. It's a hack tho.